### ISSUE TITLE
my iotas are pending 5 days - i had proofed the hash - i think the reveive adress is wrong - perhaps a mistake ?

### ISSUE THREAD

**Issue by alfredl1** [iotaledger/wallet#159]
Dear Iota support tema I 
i had purchased Iota coins on BitFinxx Exchange on July 9th. 
i had create my IOTA wallet 1 month before and i have send iotas 3 weeks before - with surpise ! 
and now i made the same (create a new adress) and the withdraw from bitfinex are comfirmed on the July 9th (i send you a screenshot) 
My IOAT Coins (972mi) are still pending in my Iota wallet at the July 9th.  
i proofed the hash transaction and i proofed the IOATE Receive adresse and in the hash docuementation the adress is diferent to the adress i had GENERATE (look at he screenshot) - perhaps there is the mistake ?

Can you please provide assistance to have the coins transferred from pending to complete and place in the balance of my Iota wallet.

Thank you.

Best Regards,

Friedrich Haselgruber

in the message i send you 1 PDF with the Hash, the Receive Adress from my IOAT Wallett, the comfirmed 

[pending_IOTA.pdf](https://github.com/iotaledger/wallet/files/1149170/pending_IOTA.pdf)

[pending_IOTA.pdf](https://github.com/iotaledger/wallet/files/1149173/pending_IOTA.pdf)

**Comment by wonkytonky** [comment]
seems the transsaction never got confirmed.. write  support bitfinex and let them reattach it. 
http://iota.tips/search/?kind=transaction&hash=NVJBTRZZLFFQNV9MULD9QZSIKMWPVKGQVABQXCGWRENNNJHSCUQJLU9PPEKIVYPUIGPQXCXMVWOE99999

I assume this is the transaction from bitfinex to you! . it's not confirmed so bitfinex support  should be able to fix it.

**Comment by wonkytonky** [comment]
i just sendt you 1 iota.. check if it arrived :)

**Comment by alfredl1** [comment]
thank you very much - i will quick tomorrow and give you feedback !

i have sent IOTA before and it was all OK !

Thank you very much - i give you feedback tomorrow !

kind regards

**Comment by alfredl1** [comment]
hi barbar !

the 1 i IOTA is arrieved in my IOAT Wallett !
question: you use this receive adress - if bitfinex will send me my IOTA
once more i must create a new receive adress - correctly ?

summary: we can say that the mistake is by bittfinex - correctly ?

zhank you very much - kind regards

2017-07-14 21:26 GMT+02:00 Friedrich Haselgruber <
f.haselgruber1961@gmail.com>:

**Comment by wonkytonky** [comment]
you can accept multiple times on the same adress , 
only if you SEND  yourself .; it's advised to generate  new adress. 

tldr: bitfinex didnt make a mistake.;  they did send iotas to you  but it never got confirmed .so now they should resend it too you  after you speak with them in support. 

they can double check like i did if they send didnt confirm so you should be fine the moment they resend it. 
regards

**Comment by alfredl1** [comment]
i wrote the bittfinex support yesterday but no answer from bittfinex !
in the hash details are a diferent receive adress from my iota wallet -
perhaps thats the mistake ?
we will see when the support is answer :)
regards

2017-07-15 14:44 GMT+02:00 barbar <notifications@github.com>:

**Comment by wonkytonky** [comment]
it often takes a few days for support to react. 

if the reciever address is different then your own wallet adress you filled in the wrong reciever address?

**Comment by wonkytonky** [comment]
btw there's a slack chat where you can ask all these questions and people probably will respond faster and have more knowledge then me

**Comment by alfredl1** [comment]
thanks barber !

by BITFINEX i filled the right receive adress - i can the in the withdraw but if i klick on the receive dress link there will be an different address and in the hash documentation is also a different receive address !

perhaps thats the mistake ?

i will wate for the answer from bittfinex !

In the meantime I do not buy IOTA more because if the API is faulty I send my money somewhere.

Thank you very much !
